src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonDocumentTests.cs

```diff
@@ -692,6 +692,7 @@ public static void ParseSimpleObject()
                 Assert.Equal("1 Microsoft Way", street);
                 Assert.Equal("Redmond", city);
                 Assert.Equal(98052, zip);
+                Assert.True(parsedObject.GetPropertyCount() > 0);
```

Comment:
Could this be asserting on a specific count? Or is it variable?

Reply:
updated